File: MyCustomView/app/src/main/java/org/techtown/customview/MainActivity.java
package org.techtown.customview;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        CustomView view = new CustomView(this);
        setContentView(view); //이 화면에 채워질 최상위 뷰

    }
}

File: androidFunction/koin/app/src/main/java/com/example/android/koin/view/MainActivity.kt
package com.example.android.koin.view

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.example.android.koin.R
import com.example.android.koin.viewmodel.MyViewModel
import org.koin.android.viewmodel.ext.android.viewModel

class MainActivity : AppCompatActivity() {

    val myViewModel : MyViewModel by viewModel()

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

    }
}